
I have no locales with UTF-8 or utf-8 ...

  - Rene Rebe <rene@exactcode.de>

--- less-382/charset.c.vanilla	2005-03-24 01:15:59.000000000 +0100
+++ less-382/charset.c	2005-03-24 01:18:57.000000000 +0100
@@ -251,7 +251,8 @@
 	    (s = lgetenv("LC_CTYPE")) != NULL ||
 	    (s = lgetenv("LANG")) != NULL)
 	{
-		if (strstr(s, "UTF-8") != NULL || strstr(s, "utf-8") != NULL)
+		if (strstr(s, "UTF8") != NULL || strstr(s, "utf8") != NULL ||
+		    strstr(s, "UTF-8") != NULL || strstr(s, "utf-8") != NULL)
 			if (icharset("utf-8"))
 				return;
 	}
